src/components/shared/translations.js

```diff
@@ -84,24 +103,51 @@ const de = {
   operations: {
     abort: "Abbrechen",
     add: 'Hinzufügen',
+    close: 'einklappen',
     delete: 'Löschen',
     deselect: {
       all: 'Auswahl aufheben'
     },
+    insert: 'Einfügen',
     new: 'Neu',
     none: 'Keine',
+    remove: 'Entfernen',
     save: "Speichern",
     select: {
       all: 'Alle auswählen',
       placeholder: 'Bitte wählen Sie einen Eintrag aus.'
     },
+    open: 'ausklappen',
     update: 'Aktualisierung'
   },
+  pager: {
+    amountMultipleOf: 'von',
+    amountMultipleLabel: 'Wählen Sie, wieviele von insgesamt {results} {items} pro Seite ausgegeben werden sollen.',
+    amountMultipleItems: 'Einträgen',
+    gotoPageFromPages ({ page, total }) {
+      return `Gehe zu Seite ${page} von ${total}`
+    },
+    label: 'Seitennavigation',
+    next: 'Nächste Seite',
+    pageFromPagesCurrentPage ({ page, total }) {
+      return `Seite ${page} von ${total}, aktuelle Seite`
+    },
+    perPage: 'Einträge anzeigen',
```

Comment:
Is this the correct translation? The key does not seem to be related

Reply:
Yes in `messages+intl-icu.de.yml` it's `pager.per.page: "Einträge anzeigen"`
Maybe `showEntries : 'Einträge anzeigen'` better!

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/918/commits/6a98cd9345f237940dc05972a9da81c10d0eb892